h2o-py/tests/testdir_algos/coxph/pyunit_coxph_smoke.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.coxph import H2OCoxProportionalHazardsEstimator
```

Comment:
I did a small check if this is correctly exposed in `h2o.estimators` as the rest of the algos. It is. So just doing `from h2o.estimators import H2OCoxProportionalHazardsEstimator` works. :heavy_check_mark: